### PR TITLE
Proposes fix to Synaptics touchpad not working after sleep wake

### PR DESCRIPTION
**Pull request by KDERazorback**
**Changes on this commit:**
+ Added Vendor Specific Message with code **151 (0x97)** to `ApplePS2Device` used to request Trackpad resets from Keypad controller.
- `VoodooPS2Keyboard` now requests a Trackpad reset too when the 3-finger salute (_Ctrl+Alt+Del_) is pressed on the Keypad.
+ Fix: Added an extra `kDP_Enable` command request on `VoodooPS2SynapticsTouchPad` when waking from sleep. This should auto-enable reluctant Synaptics touchpads on wake.
+ Added handling code for resetTouchpad IOkit messages on `VoodooPS2SynapticsTouchPad`, to actually reset and re-enable the device on request.
- Bumped module version to **2.1.6**

**TL;DR:** This commit tries to fix some **Synaptics touchpad devices** not working after waking from sleep. If that doesnt work automatically after a couple of seconds, press **Ctrl+Alt+Del to issue a device reset**. This can be done at any time.

**Dev note:** It may be a good idea to implement the the reset message handling on any other touchpad driver too, this should give users a keyboard combo command to try to recover from non-working scenarios, before giving up and rebooting.

### CODE REVIEW

**Comment by usr-sse2** [comment]
Ctrl-Alt-Del may be used for virtual machines and remote desktops, so it's not a good idea to intercept this key combination. Currently we use Print Screen as a special key (PrtSc – disable or enable touchpad, Cmd+PrtSc – disable or enable both keyboard and touchpad). Maybe Cmd-Opt-PrtSc?
Ideally, all three key combinations should be configurable from Info.plist or SSDT.

**Comment by KDERazorback** [comment]
> Ctrl-Alt-Del may be used for virtual machines and remote desktops

True, but also consider this key combination is already intercepted internally on **VoodooPS2Keyboard.cpp** on the `bool ApplePS2Keyboard::dispatchKeyboardEventWithPacket(const UInt8*)` method around line `1581` to actually generate a fake **ADB 0x7f (power button)** signal. It was added back by RehabMan to invoke the "Shutdown/Reboot/Sleep" dialog on the system and never got removed despite being broken since ancient years.

> Ideally, all three key combinations should be configurable from Info.plist or SSDT.

Agreed, but i think that would require a different PR/commit. I can do that change but since it will directly depend on this PR, this should be accepted first before implementing the Info.plist/SSDT part or even removing the old RehabMan code.

**Comment by KDERazorback** [comment]
I've updated the commit to reflect the use of the **PrntScr** key as the primary hotkey for **Touchpad reset commands**. Also improved modifier key handling on those scenarios where that key is used. 

Please review the commit changes and feel free to give any other feedback.

I strongly suggest to remove the **Ctrl+Alt+Del** code left by RehabMan, since it no longer makes sense, or at least we could try to implement it properly. This could be done in a latter commit along with the SSDT configuration options for special command handling, but those changes are beyond the scope of this PR.
